README.md

```diff
@@ -54,6 +54,7 @@ Currently supported grammars are:
 | LiveScript                           | Yes        | Yes             | |
 | Lua                                  | Yes        | Yes             | |
 | Makefile                             | Yes        | Yes             | |
+| [MATLAB](http://mathworks.com/products/matlab) | Yes        | Yes             | Requires the language-matlab package in Atom https://atom.io/packages/language-matlab |
```

Comment:
Looks like there is nothing special in language-matlab package. For grammar support we have a `NOTE` section at the bottom of the table.

Did i miss something and this package is necessary not only fo grammar?